R/sof.mpm2.R

```diff
@@ -60,25 +61,31 @@ makeMPM2Function = function(n.peaks, dimensions, topology, seed, rotated = TRUE,
   # build parameter set (bounds are [0, 1]^d)
   par.set = makeNumericParamSet("x", len = dimensions, lower = 0, upper = 1)
 
-  # import rPython namespace
-  BBmisc::requirePackages("_rPython", why = "smoof::makeMultiplePeaksModel2Function")
+  # import reticulate namespace
+  BBmisc::requirePackages("_reticulate", why = "smoof::makeMultiplePeaksModel2Function")
+
+  # FIXME: initialize 3 functions from mpm2.py as NULL such that they have a visible binding when checking the pkg
+  evaluateProblem = getGlobalOptimaParams = getLocalOptimaParams = NULL
 
   # load funnel generator to global environemt
-  eval(rPython::python.load(system.file("mpm2.py", package = "smoof")), envir = .GlobalEnv)
+  eval(reticulate::py_run_file(system.file("mpm2.py", package = "smoof")), envir = .GlobalEnv)
+  topol = topology
```

Comment:
`topol` is never used after this assignment.

Reply:
good point, I removed it in the updated version